storage/Devices/Disk.cc

```diff
@@ -34,7 +36,7 @@ namespace storage
 
 
     Disk::Disk(Impl* impl)
-	: BlkDevice(impl)
+	: Partitionable(impl)
```

Comment:
Nitpicking: http://dictionary.reference.com/browse/partitionable is a not a noun but an adjective